astronomer/providers/dbt/cloud/sensors/dbt.py

```diff
@@ -0,0 +1,61 @@
+import time
+from typing import Any, Dict, Optional
+
+from airflow import AirflowException
+from airflow.providers.dbt.cloud.sensors.dbt import DbtCloudJobRunSensor
+from airflow.utils.context import Context
+
+from astronomer.providers.dbt.cloud.triggers.dbt import DbtCloudRunJobTrigger
+
+
+class DbtCloudJobRunSensorAsync(DbtCloudJobRunSensor):
+    """
+    Checks the status of a dbt Cloud job run.
+
+    .. seealso::
+        For more information on how to use this sensor, take a look at the guide:
+        :ref:`howto/operator:DbtCloudJobRunSensor`
```

Comment:
same here, please check in docs if it work

Reply:
Yes, it was working I have mentioned the sync version of sensor

Reply:
![Uploading Screenshot 2022-09-10 at 10.37.07 AM.png…]()